### ISSUE TITLE
GTA V Folder not being found

### ISSUE THREAD

**Issue by Rubylium** [vscorpio/gta5rpf-to-fivem#16]
I do select the right folder, but the status still say not found.
GTA V has been downloaded from Epic Game Store, maybe that's the issue ?

![image](https://user-images.githubusercontent.com/19718604/104030521-d2190c80-51cb-11eb-8ad2-9b5117389e87.png)

**Comment by Avenze** [comment]
yeah, that's the issue, I solved that issue and merged into this repo, you can find the build that has the epic games support in my other repository, refer to this issue for the link, https://github.com/vscorpio/gta5rpf-to-fivem/issues/11#issuecomment-739553556
